### DISCUSSION TITLE
Issue with domain configuration en dev environment (localhost)

### DISCUSSION THREAD

**Discussion by rtouze** [laravel/nova-issues#3245] [Q&A]
- Laravel Version: 8.28.1
- Nova Version: 3.22.0
- PHP Version: 7.4.14
- Database Driver & Version:
- Operating System and Version: 5.4.0-65-generic #73~18.04.1-Ubuntu SMP Tue Jan 19 09:02:24 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

### Description:

I upgraded from 3.12.1 to 3.22 yesterday and my local / dev domain configuration do not work anymore.

In 3.12.1, configuration bellow worked fine:

```
app.url = 'http://localhost:8000'
nova.domain = 'admin.localhost'
nova.path = '/'
```
Now it give me a 404 when accessing `http://admin.localhost:8000`
Setting nova.domain to null is working though.
I've seen that you changed domain handling in 3.16.x, so I tried to set nova.domain to `admin.localhost:8000` as it's needed now to activate NovaServiceProvider (I checked with log that it is actually activated when I add the port to nova.domain), but I still have the 404 for `http://admin.localhost:8000`.

Are there remaining issues on this or do I miss something ?

**Comment by crynobone** [comment]
You should use `nova.domain = 'admin.localhost:8000'` in this case.

**Reply by rtouze** [reply]
Thank you. Unfortunately, it does not work either. I can see that NovaServiceProvider is activated but I still face a 404.
